exercises/pascals-triangle/description.md

```diff
@@ -4,10 +4,11 @@ In Pascal's Triangle each number is computed by adding the numbers to
 the right and left of the current position in the previous row.
 
 ```text
-    1
-   1 1
-  1 2 1
- 1 3 3 1
-1 4 6 4 1
+     1
+    1 1
+   1 2 1
+  1 3 3 1
+ 1 4 6 4 1
+1 5 10 10 5 1
```

Comment:
the alignment between this row and the one above is no longer effective because 10 has two digits